docs/index.md

```diff
@@ -80,8 +80,8 @@ local myName = Value("Daniel")
 
 -- This creates a state object from a calculation.
 -- It determines its own value automatically.
-local myGreeting = Computed(function()
-	return "Hello! My name is " .. myName:get()
+local myGreeting = Computed(function(use, scope)
+	return "Hello! My name is " .. use(myName)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
local myGreeting = Computed(function(use)
```

Reply:
This should reflect `v3.0` with **scope**
```suggestion
local myGreeting = scope:Computed(function(use)
```

Reply:
done